microcosm_pubsub/dispatcher.py

```diff
@@ -74,6 +65,18 @@ def handle_message(self, message, bound_handlers):
             )
             return result
 
+    def find_handler(self, message, bound_handlers):
+        media_type = message.media_type
+        content = message.content
+
+        if content is None:
```

Comment:
I don't see why the first error case should be in `find_handler` function. It should live in a separate assertion function